documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p><code>Convert the values stored in the Survived column to a list of unique values. Survived has one of two values: 0 or 1.</code></p><p> </p><p>Also, what's this for? Just to show the user the labels? "Convert the values..." plus being in the "prepare" section indicates that this is part of data pre-processing, yet <code>classes</code> isn't used except for this cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>